tests/conftest.py

```diff
@@ -148,9 +148,8 @@ def setup(input_data, base=mock_backend.Base):
             Session.configure(bind=engine)
             session = Session()
             # Load test data
-            for entity in iter_flatten(input_data):
-                session.add(entity)
-                session.commit()
+            session.bulk_save_objects(iter_flatten(input_data))
+            session.commit()
```

Comment:
Great. I tried optimizing this at one point and couldn't get a good speed-up. I didn't find `bulk_save_objects()`, though.